lib/resources/nginx_conf.rb

```diff
@@ -82,10 +82,13 @@ def resolve_references(data, rel_path)
       # Any call to `include` gets its data read, parsed, and merged back
       # into the current data structure
       if data.key?('include')
-        data.delete('include').flatten
-            .map { |x| File.expand_path(x, rel_path) }
-            .map { |path| parse_nginx(path) }
-            .map { |e| data.merge!(e) }
+        include_paths = data.delete('include').flatten.map do |path|
+          paths = inspec.command("ls #{File.expand_path(path, rel_path)}").stdout.split
```

Comment:
Potentially, or we find a solution that doesn't require this command at all. I'd treat it as an investigation, we might need a pattern for this down the line :)